package-lock.json

```diff
@@ -1,15708 +1,8 @@
 {
   "name": "@asyncapi/parser",
   "version": "2.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
You are using an old version of node/npm. Please restore this file to previous version.

Reply:
Done.